docs/extras/integrations/callbacks/labelstudio.ipynb

```diff
@@ -0,0 +1,335 @@
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "You'll need an `API_KEY` to make API calls. Open your LabelStudio instance in your browser, go to `Account & Settings > Access Token` and copy the key.\n",
+    "\n",
+    "Set environment variables with your LabelStudio URL and API key:"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ['LABEL_STUDIO_URL'] = '<YOUR-LABEL-STUDIO-URL>'  # e.g. http://localhost:8080\n",
+    "os.environ['LABEL_STUDIO_API_KEY'] = '<YOUR-LABEL-STUDIO-API-KEY>'"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now you can use the `LabelStudioCallbackHandler` in your LangChain pipeline."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Collecting LLMs prompts and responses"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "You can collect input LLM prompts and output responses in a LabelStudio project, allowing subsequent editing and rating of the responses:"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "from langchain.llms import OpenAI\n",
+    "from langchain.callbacks import LabelStudioCallbackHandler\n",
+    "\n",
+    "llm = OpenAI(\n",
+    "    temperature=0,\n",
+    "    callbacks=[LabelStudioCallbackHandler(project_name=\"My Project\")]\n",
+    ")\n",
+    "print(llm(\"Tell me a joke\"))"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "After running this code, open LabelStudio and navigate to the project named `My Project`. You will see the prompts, responses, and metadata like the model name.\n",
+    "\n",
+    "This allows consolidating the prompts and responses in LabelStudio for easier reviewing, labeling, and iteration. The project persists the data and provides an interface for human evaluation and feedback."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Collecting Chat model Dialogues"
```

Comment:
This will fail without a project ID. We should talk about how to discover and set this value.

Also for my environment, I'm getting a 400 error with that call:

```
Request URL: http://localhost:8080/api/projects/2/import?return_task_ids=1
Response status code: 400
Response content:
{
  "id": "0517f77d-1478-42c7-89f4-fa69b0e03962",
  "status_code": 400,
  "version": "1.8.1",
  "detail": "Validation error",
  "exc_info": null,
  "validation_errors": {
    "non_field_errors": [
      "Error at item 0: data['prompt']=[{'role': 'SystemMessage', 'content': 'Always use a lot of emojis'}, {'role': 'HumanMessage', 'content': 'Tell me a joke'}] is of type 'list', but the object tag Text expects the following types: ['str', 'int', 'float'] [assume: item[\"data\"] = task root with values] :: {'data': {'prompt': [{'role': 'SystemMessage', 'content': 'Always use a lot of emojis'}, {'role': 'HumanMessage', 'content': 'Tell me a joke'}], 'run_id': '8f066013-fe39-4b2e-9e1b-5c47ef21827c'}, 'predictions': [{'result': [{'from_name': 'response', 'to_name': 'prompt', 'type': 'textarea', 'value': {'text': [\"Sure, here's a joke for you: \ud83c\udf3d Why did the scarecrow win an award? Because he was outstanding in his field! \ud83c\udf3e\ud83c\udfc6\"]}}], 'model_version': 'gpt-3.5-turbo'}]}"
    ]
  }
}

SDK error traceback:
Error in LabelStudioCallbackHandler.on_llm_end callback: 400 Client Error: Bad Request for url: http://localhost:8080/api/projects/2/import?return_task_ids=1
```